fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/api/event/registry/DynamicRegistrySetupCallback.java

```diff
@@ -29,13 +29,18 @@
  * <pre>
  * {@code
  * DynamicRegistrySetupCallback.EVENT.register(registryManager -> {
- *     Registry<Biome> biomes = registryManager.get(Registry.BIOME_KEY);
- *     RegistryEntryAddedCallback.event(biomes).register((rawId, id, object) -> {
- *         // Do something
+ *     registryManager.getOptional(Registry.BIOME_KEY).ifPresent(biomes -> {
+ *         RegistryEntryAddedCallback.event(biomes).register((rawId, id, object) -> {
+ *             // Do something
+ *         });
  *     });
  * });
  * }
  * </pre>
+ *
+ * <p><strong>Important Note</strong>: The passed dynamic registry manager might not
+ * contain the registry. Use {@link DynamicRegistryManager#getOptional} to
```

Comment:
This is due to the multiple layers, right? In that case, it would be good to explain this somewhere.